app/javascript/src/components/Dashboard/Notes/FormPane/CreateForm.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { Input, Textarea, Select } from "neetoui/v2";
+
+const CreateForm = () => (
+  <form className="w-full px-2 space-y-6">
+    <Input
+      label="Title"
+      name="title"
+      placeholder="Enter note title"
+      className="flex-grow-0 w-full"
+      required
```

Comment:
Implicitly true attrs should be written at the top of the component's attr list.